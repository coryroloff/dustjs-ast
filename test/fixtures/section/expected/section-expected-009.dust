{#section key=value newKey=three.two.one}{/section}