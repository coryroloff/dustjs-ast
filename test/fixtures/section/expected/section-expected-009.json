{
	"type": "Template",
	"body": [
		{
			"type": "Section",
			"body": [],
			"params": [],
			"format": {
				"startTagAfterStart": null,
				"startTagBeforeParam": [],
				"startTagBeforeEnd": null,
				"endTagAfterStart": null,
				"endTagBeforeEnd": null
			},
			"kind": "?",
			"id": {
				"type": "Identifier",
				"path": [
					{
						"type": "Key",
						"name": "exists",
						"start": 2,
						"end": 8,
						"loc": {
							"start": {
								"line": 1,
								"column": 2
							},
							"end": {
								"line": 1,
								"column": 8
							}
						}
					}
				],
				"localContext": false,
				"start": 2,
				"end": 8,
				"loc": {
					"start": {
						"line": 1,
						"column": 2
					},
					"end": {
						"line": 1,
						"column": 8
					}
				}
			},
			"start": 0,
			"end": 10,
			"loc": {
				"start": {
					"line": 1,
					"column": 0
				},
				"end": {
					"line": 1,
					"column": 10
				}
			}
		}
	],
	"tokens": [
		{
			"type": "Punctuator",
			"value": "{",
			"start": 0,
			"end": 1,
			"loc": {
				"start": {
					"line": 1,
					"column": 0
				},
				"end": {
					"line": 1,
					"column": 1
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "?",
			"start": 1,
			"end": 2,
			"loc": {
				"start": {
					"line": 1,
					"column": 1
				},
				"end": {
					"line": 1,
					"column": 2
				}
			}
		},
		{
			"type": "Key",
			"value": "exists",
			"start": 2,
			"end": 8,
			"loc": {
				"start": {
					"line": 1,
					"column": 2
				},
				"end": {
					"line": 1,
					"column": 8
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "/",
			"start": 8,
			"end": 9,
			"loc": {
				"start": {
					"line": 1,
					"column": 8
				},
				"end": {
					"line": 1,
					"column": 9
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "}",
			"start": 9,
			"end": 10,
			"loc": {
				"start": {
					"line": 1,
					"column": 9
				},
				"end": {
					"line": 1,
					"column": 10
				}
			}
		}
	],
	"start": 0,
	"end": 10,
	"loc": {
		"start": {
			"line": 1,
			"column": 0
		},
		"end": {
			"line": 1,
			"column": 10
		}
	}
}